internal/pkg/admission/policy-server-deployment.go

```diff
@@ -329,22 +329,9 @@ func (r *Reconciler) deployment(configMapVersion string, policyServer *policiesv
 			},
 		)
 	}
-	enableReadOnlyFilesystem := true
-	privileged := false
-	runAsNonRoot := true
-	allowPrivilegeEscalation := false
-	capabilities := corev1.Capabilities{
-		Add:  []corev1.Capability{},
-		Drop: []corev1.Capability{"all"},
+	if policyServer.Spec.Security.Container != nil {
```

Comment:
I would prefer to not drop the explicit defaults we had before. The code should be something like that:

```go
if policyServer.Spec.Security.Container != nil {
  // use whatever is defined inside of the CRD
} else {
  // the previous code
}
```

Reply:
I would also like this.